Firestore/src/Query.php

```diff
@@ -195,6 +195,53 @@ public function count(array $options = [])
         return $aggregationResult->get('count');
     }
 
+    /**
+     * Gets the sum of all documents matching the provided query filters.
+     *
+     * Example:
+     * ```
+     * $sum = $query->sum();
+     * ```
+     * @param string $field The relative path of the field to aggregate upon.
+     * @param array $options [optional] {
+     *     Configuration options is an array.
+     *
+     *     @type Timestamp $readTime Reads entities as they were at the given timestamp.
+     * }
+     * @return int
```

Comment:
Q: Will this be always int?
What is the field which is summed up, is non-numeric?
Such as one of https://cloud.google.com/firestore/docs/reference/rest/v1/Value?

Reply:
This is `mixed` in AggregateQuerySnapshot

Reply:
Fixed